nominal/__init__.py

```diff
@@ -39,4 +41,6 @@
     "Attachment",
     "NominalClient",
     "Video",
+    "LogSet",
+    "Log",
```

Comment:
need to add these to the filter in docs/reference/toplevel.md

Reply:
ah sorry, I thought this _was_ the filter